docs/modules/ROOT/pages/partials/con_hermetic_verification.adoc

```diff
@@ -1,16 +1,16 @@
 .Verification
 
-. On your application, go to the *Activity > Pipeline runs* tab.
+. From the *Applications* view, go to *Activity > Pipeline runs*.
 
 ifdef::myfunctionone[]
 
-. Go to the pipeline run with *Type* as *Build* and confirm the `build-container` stage displays a green check mark indicating that the build process has successfully fetched all the dependencies.
+. Look at the pipeline run with *Build* in the *Type* column and confirm that the `build-container` stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.
 
 endif::[]
 
 ifdef::myfunctiontwo[]
 
-. For a pipeline run with *Type* as *Build* and confirm the `pre-fetch dependencies` stage displays a green check mark indicating that the build process has successfully pre-fetched all the dependencies.
+. Go to the pipeline run with *Build* in the *Type* column and confirm that the `pre-fetch dependencies` stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.
```

Comment:
If they followed the procedure and updated both push and pull definition, wouldn't there be two builds running, one for each? What happens if it isn't green? How likely is it that they'll build successfully the first try? Do we need a quick note on where they should look if it failed, and how to kick off a new build once they've tried to fix it?

Reply:
@chmeliik can you answer Matt's questions above?

Reply:
> If they followed the procedure and updated both push and pull definition, wouldn't there be two builds running, one for each?

The push definition runs for pushes to the main branch (or whatever branch the component tracks) - i.e. it runs after the PR is merged. Only the pull-request definition runs when the PR is opened / changed.

> What happens if it isn't green? How likely is it that they'll build successfully the first try? Do we need a quick note on where they should look if it failed, and how to kick off a new build once they've tried to fix it?

First they should probably check the build logs. If it's not obvious how to find them in the UI it might be worth mentioning. A new build would start automatically as soon as they've made changes in the PR

Reply:
thanks @chmeliik. i'm reluctant to add that, in the case of a build failure, they should check the build logs unless i can tell them clearly where to find those, what to do next, etc. can you easily tell me how to tell users where to find build logs?

Reply:
no, @mattreid your instincts are spot on. we should always allow for the possibility of failure so that we never leave users high and dry, regardless of likelihood. lemme make some adds based on what you Slacked me. stay tuned